main.tf

```diff
@@ -108,6 +108,7 @@ resource "google_vpc_access_connector" "main" {
   network        = google_compute_network.main.name
   region         = var.region
   max_throughput = 300
+  min_throughput = 200
```

Comment:
We encountered the following error in integration tests:
```
Error encountered:
Error 400: Connector minimum throughput must be a multiple of 100 between 200 and 1000 in Mbps.
```

Even though the docs state that min_throughput defaults to 200, it doesn't seem like it does.
Docs: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/vpc_access_connector#min_throughput-1